app/services/homepage/fetch_tag_flares.rb

```diff
@@ -0,0 +1,33 @@
+module Homepage
+  class FetchTagFlares
```

Comment:
If we're not meant to instantiate this, should it be a module instead?

```suggestion
  module FetchTagFlares
```

Reply:
Good point!